unime/src-tauri/tests/fixtures/states/shenron_six_credentials_two_connections.json

```diff
@@ -0,0 +1,220 @@
+{
+    "profile_settings": {
+        "locale": "en-US",
+        "profile": {
+            "name": "Shenron",
+            "picture": "&#x1F432",
+            "theme": "dark",
+            "primary_did": "did: key: z6MkvL5rMcDmrzzAWaFuhoHcJ4TKCHtTF6MjknkWgyZZBQar"
```

Comment:
Did you search & replace `:` for `: ` instead of formatting?

Reply:
So `did:key:z6MkvL5r` now became `did: key: z6MkvL5r`

Reply:
yess i did, see my reply to previous comment haha